pkg/mock/client-go/discovery/mocks.go

```diff
@@ -6,7 +6,7 @@ package discovery
 
 import (
 	gomock "github.com/golang/mock/gomock"
-	openapi_v2 "github.com/googleapis/gnostic/OpenAPIv2"
+	OpenAPIv2 "github.com/googleapis/gnostic/OpenAPIv2"
```

Comment:
Please revert this change, this file is generated and CI fails here.

Reply:
Sorry, mockgen was outdated